src/poliastro/threebody/cr3bp_char_quant.py

```diff
@@ -0,0 +1,236 @@
+"""
+@author: Dhruv Jain, Multi-Body Dynamics Research Group, Purdue University
+
+Useful ideas:
+1. To non-dimensioanlize a physical position vector: divide by l*
+2. To dimensioanlize a [nd] position vector: multiply by l*
+3. To non-dimensionalize a physical veloctiy vecotr, multiply by t*/l*
+4. To dimensionalize a [nd] veloctiy vecotr, multiply by l*/t*
+5. Position of P1 = [-mu, 0, 0] [nd] {x,y,z}
+5. Position of P2 = [1-mu, 0, 0] [nd] {x,y,z}
+"""
+from astropy import units as u
+
+import poliastro.constants.general as general
+import poliastro.constants.mean_elements as dist
+
+
+class SystemChars:
+    """Computes and stores the properties (mu, l*, t*) of a CR3BP system
+    'mu': mass ratio of P1-P2 primary bodies
+    'l*': characterisitic lenght of P1-P2 primary bodies
+    't*': characterisitic time of P1-P2 primary bodies
+    If P2 is more massive than P1 then swap the Pi, so that P1 is the more massive body
+    """
+
+    def __init__(self, p1="Earth", p2="Moon"):
+        """
+        Constructor
+
+        Parameters
+        ----------
+        p1 : string
+           'body_i'. The deafult is Earth
+        p2 : string
+           'body_j'. The deafult is Moon
+        """
+
+        mu, lstar, tstar, p1, p2 = self.BodiesCharCompute(
+            p1.lower(), p2.lower()
+        )  # lower() to ensure lowercase characters
+        units_nd = u.def_unit(
+            "[nd]"
+        )  # Define custome units, [nd]: non-dimensional unit
+
+        self._mu = mu * units_nd
+        self._lstar = lstar
+        self._tstar = tstar
+        self._p1 = p1
+        self._p2 = p2
```

Comment:
A pattern we're using in poliastro a lot:

- Have `__init__` methods in classes just store variables:

```suggestion
    def __init__(self, lstar, tstar):
        """
        Constructor

        Parameters
        ----------
        mu
        lstar
        tstar
        """

        self._mu = mu * units_nd
        self._lstar = lstar
        self._tstar = tstar
```

(unsure if the two primaries are needed)

- Add `@classmethod`s to create the class:

```python
@classmethod
def from_primaries(cls, p1, p2):
    mu, lstar, tstar, p1, p2 = self.BodiesCharCompute(
        p1.lower(), p2.lower()
    )  # lower() to ensure lowercase characters
    units_nd = u.def_unit(
        "[nd]"
    )  # Define custome units, [nd]: non-dimensional unit
    return cls(mu, lstar, tstar)
```

(see my comment about `p1` and `p2` possibly being `Body` objects)
(see my above to my question about having `p1` and `p2` in the class itself)
(see my other comment about units)

Reply:
I will work on making the p1 and p2 as Body objects then go from there.